socorro/cron/jobs/fetch_adi_from_hive.py

```diff
@@ -238,7 +240,7 @@ class FetchADIFromHiveCronApp(BaseCronApp):
     @staticmethod
     def remove_control_characters(s):
         if isinstance(s, str):
-            s = unicode(s, 'utf-8', errors='replace')
+            s = str(s, 'utf-8', errors='replace')
```

Comment:
I'd much rather we were using `six`. Otherwise this looks totally wrong in Python 2 which is our primary context.

In six, this would be:

```
s = six.text_type(s, 'utf-8', errors='replace')
```